spring-cloud-gcp-autoconfigure/pom.xml

```diff
@@ -78,16 +78,6 @@
             <groupId>com.google.cloud.sql</groupId>
             <artifactId>mysql-socket-factory</artifactId>
             <optional>true</optional>
-            <exclusions>
-                <exclusion>
-                    <groupId>mysql</groupId>
```

Comment:
We might still need this one?

Reply:
This dependency is now marked as `provided`. So, I don't think we need to exclude it.
https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/blob/28b266b13ec4a9b6204226a864453d3903095f7e/connector-j-5/pom.xml#L25